src/theory/arith/theory_arith.cpp

```diff
@@ -41,7 +41,6 @@ TheoryArith::TheoryArith(context::Context* c,
       d_internal(
           new TheoryArithPrivate(*this, c, u, out, valuation, logicInfo, pnm)),
       d_ppRewriteTimer("theory::arith::ppRewriteTimer"),
-      d_proofRecorder(nullptr),
```

Comment:
Great! This is an example of a reference to the `src/proof` tree, and, thus, it should be removed :+1: .